### DISCUSSION TITLE
Bug Report: Can't reach [name]!

### DISCUSSION THREAD

**Discussion by lewri** [FakeFishGames/Barotrauma#15266] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I noticed while having 3 people follow me around in single player the medic was unable to reach one. However, instead of using their name the radio text used the placeholder `[name]`. This did occur multiple times during 30 seconds.

![image](https://github.com/user-attachments/assets/19f0afd0-60ee-49dc-a35d-77bafc6f95d5)

### Reproduction steps

1. Have 3 people follow you in the sub
2. Somehow cause injuries to the party
3. Wait for the medic. Though I think mine occurred while moving around a bit still.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in a later update. Closing discussion.
